### PR TITLE
Add Work In Progress Annotation #216

### PR DESCRIPTION
**Pull request by oomelianchuk**

### CODE REVIEW

File: src/main/java/com/xceptance/neodymium/NeodymiumRunner.java [hunk 1]
```diff
@@ -200,9 +200,15 @@ protected List<FrameworkMethod> computeTestMethods()
         // from methodBlock
         List<Class<? extends StatementBuilder>> statementRunOrder = new DefaultStatementRunOrder().getRunOrder();
 
+        List<FrameworkMethod> computedMethods = super.computeTestMethods();
+        boolean wipMethod = computedMethods.stream().anyMatch(computedMethod -> computedMethod.getAnnotation(WIP.class) != null);
         // super.computeTestMethods will return all methods that are annotated with @Test
```
Comment (wurzelkuchen) [inline src/main/java/com/xceptance/neodymium/NeodymiumRunner.java:205]: Comment describes a line of code that is not below it any more, but on line 203 now. Needs to get shifted upwards a bit

File: src/main/java/com/xceptance/neodymium/WIP.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Retention(RUNTIME)
+@Target(
+{
+  METHOD
+})
+public @interface WIP
```
Comment (wurzelkuchen) [inline src/main/java/com/xceptance/neodymium/WIP.java:13]: JavaDoc

**Review by wurzelkuchen — CHANGES_REQUESTED** [review]
I'd like to have a configuration property, which enables/disabled this feature and is default on false. 
Background: introducing something like this brings the danger to accidentally commit it to a real test build and it might not get noticed. Adding an additional barrier, which can be used in the dev-neodymium properties might help prevent such incidents.

File: src/main/java/com/xceptance/neodymium/NeodymiumRunner.java (hunk 1 above)
Comment (kqmpetenz) [reply src/main/java/com/xceptance/neodymium/NeodymiumRunner.java:205]: done

File: src/main/java/com/xceptance/neodymium/WIP.java (hunk 2 above)
Comment (kqmpetenz) [reply src/main/java/com/xceptance/neodymium/WIP.java:13]: done

**Review by wurzelkuchen — APPROVED** [review]

**Pull request merged**